script/upload_to_aptly

```diff
@@ -130,10 +130,10 @@ done
 for i in ${DIST}; do
     echo Publishing distro ${i}
     if [ "${i}" = "stable" ]; then
-        echo Running command: aptly publish snapshot -origin="Brave Software" -label="Brave Browser" -force-overwrite=true -gpg-key="${GPG_KEY_ID}!" "${i}-snapshot" "s3:${S3_BUCKET}:"
-        aptly publish snapshot -origin="Brave Software" -label="Brave Browser" -force-overwrite=true -gpg-key="${GPG_KEY_ID}!" "${i}-snapshot" "s3:${S3_BUCKET}:"
+        echo Running command: aptly publish snapshot -batch -passphrase="xxxxxx" -origin="Brave Software" -label="Brave Browser" -force-overwrite=true -gpg-key="${GPG_KEY_ID}!" "${i}-snapshot" "s3:${S3_BUCKET}:"
```

Comment:
I don't think that passphrase does anything here since we are loading the agent and unlocking the private key previously in the run. We would also want to make sure this isn't leaking the passphrase into logs.

The batch option should help out the missing TTY though.

Reply:
Okay, it's removed in f3d3138